scripts/lib/install_opencv.sh

```diff
@@ -1,13 +1,34 @@
 #
-# Install opencv and its dependencies
+# Install opencv and related dependencies
 
 install_opencv() {
   # Update
   sudo apt-get -y update
   
   # Install pip
   sudo apt install python3-pip
+
+  # Install venv
+  sudo apt install python3-venv
+  
+  # Change To Opencv server directory
+  cd ../opencv_flask
+
+  # Create new virtual environment
+  python3 -m venv venv
+  
+  # Running the activate script
+  source venv/bin/activate
```

Comment:
I recommend doing this approach from now on when installing pip packages and not using shell script to install them individually: https://pip.pypa.io/en/stable/reference/pip_freeze/

Reply:
Created a new opencv_requirements file